ntp-daemon/src/config/server.rs

```diff
@@ -20,6 +32,10 @@ impl TryFrom<&str> for ServerConfig {
     fn try_from(value: &str) -> Result<Self, Self::Error> {
         Ok(ServerConfig {
             addr: SocketAddr::from_str(value)?,
+            denylist: IpFilter::none(),
```

Comment:
Quite a bit of duplication here, maybe add some abstraction?

Reply:
Maybe I am just too used to the code as it is, but I am not really getting what duplication you are getting at, could you clarify?

Reply:
You have multiple pairs of an `IpFilter` and a `FilterAction`. It might be better to group them in a single type.

Reply:
I have put some thought into this, and I disagree since although there is superficial duplication, the interpretation of both fields is quite significantly different between the allow and deny lists, and therefore combining these does in my opinion not really give much space for code reuse, but does give space for confusion on behavior.